nfsv4/client.c

```diff
@@ -94,6 +94,10 @@ int nfs4_pwrite(nfs4_file f, void *source, bytes offset, bytes length)
     }
 }
 
+int nfs4_write(nfs4_file f, void *source, bytes offset, bytes length) {
```

Comment:
It might be helpful to have docstrings for `nfs4_write`, `nfs4_append` and `nfs4_pwrite`, especially since `nfs4_write` is not implementing linux write which doesn't take an offset.

Reply:
added